deploy/helm/sumologic/values.yaml

```diff
@@ -11,6 +11,28 @@ sumologic:
   # If enabled, a pre-install hook will create Collector and Sources in Sumo Logic
   setupEnabled: true
 
+  # Configuration of kubernetes for terraform client
+  # https://www.terraform.io/docs/providers/kubernetes/index.html#argument-reference
+  cluster:
+    host: "https://kubernetes.default.svc"
+    # username:
+    # password:
+    # insecure:
+    # client_certificate:
+    # client_key:
+    cluster_ca_certificate: "${file(\"/var/run/secrets/kubernetes.io/serviceaccount/ca.crt\")}"
```

Comment:
Why is the double quote escaped with a backslash but not in the YAML below?

Reply:
Terraform format is not friendly:
 * cannot use `'`
 * `"` between `"` cannot be escaped (`\"` is treated as `\"` instead of `"`)

Reply:
We can use in the `values.yaml`:
 * "${file(\"/var/run/secrets/kubernetes.io/serviceaccount/ca.crt\")}"
 * or  '${file("/var/run/secrets/kubernetes.io/serviceaccount/ca.crt")}'

Reply:
AS long as that syntax works in both cases, im ok with them being slightly different :)